src/api/Base.js

```diff
@@ -123,6 +134,78 @@ class Base {
     getCache(): Cache {
         return this.cache;
     }
+
+    /**
+     * Generic success handler
+     *
+     * @param {Object} data the response data
+     * @param {Function} successCallback the success callback
+     */
+    successHandler(data: Object): void {
+        if (!this.isDestroyed() && this.successCallback && typeof this.successCallback === 'function') {
```

Comment:
Middle check unnecessary
You can also bypass the checks by defaulting to lodash/noop.

Reply:
Will remove